django/views/debug.py

```diff
@@ -635,6 +636,16 @@ def technical_404_response(request, exception):
         ):
             return default_urlconf(request)
 
+    for inner_tried in itertools.chain(*tried or []):
```

Comment:
Using an empty tuple will avoid the slight cost of creating a list in the path where `tried` is empty.
```suggestion
    for inner_tried in itertools.chain(*tried or ()):
```